docs/changelog.rst

```diff
@@ -18,6 +18,8 @@ Changelog
 * :feature:`-` An ethereum address's tokens can now be manually detected by pressing a specific button in the UI.
 * :feature:`-` Transactions involving uniswap v2 and uniswap v3 will now be properly decoded in the ethereum transactions view
 * :bug:`-` When force sync fails a proper error message is displayed
+* :bug:`-` Now if something is wrong with an asset update, it won't end up having partial information. 
```

Comment:
where are these 2 coming from and what do they have to do with savepoints?

Reply:
Partial asset info after an update is fixed in this pr.

Support of one address being on both bitcoin and bitcoin cash is coming from a fix that I made a couple of weeks ago. But realized that I didn't add a changelog entry only now.

Reply:
Isn't that already included in 1.25.3 changelog?

Reply:
Ah wow, yes indeed